### PR TITLE
Upgrade appinsights

### PR DESCRIPTION
**Pull request by timja**
https://github.com/microsoft/ApplicationInsights-Java/blob/2.5.0-BETA/CHANGELOG.md

### CODE REVIEW

File: java-logging-appinsights/build.gradle [hunk 1]
```diff
@@ -4,8 +4,7 @@ plugins {
 
 def versions = [
   springBoot: '2.1.6.RELEASE',
-  appInsights: '2.3.1',
-  appInsightsStarter: '1.1.2'
+  appInsights: '2.5.0-BETA'
```
Comment (nitinprabhu) [inline java-logging-appinsights/build.gradle]: This version Removed support for <Class> custom instrumentation in AI-Agent.xml which some of projects do use it. Also is beta version recommended for production usage ?

Comment (timja) [reply java-logging-appinsights/build.gradle]: 🤷‍♂ we need to try it out anyway.
I'll release this as beta

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+  - package_manager: "java:gradle"
```
Comment (nitinprabhu) [inline .dependabot/config.yml]: probably nice to get this in separate PR :)

Comment (timja) [reply .dependabot/config.yml]: meant to put it in my update dependencies PR to fix the owasp errors oops

Comment (timja) [reply .dependabot/config.yml]: will get the other one in first

**Pull request merged**